manifests/opendatahub/dependencies/quickstart.yaml

```diff
@@ -47,6 +47,54 @@ spec:
           configMap:
             name: etcd-scripts
             defaultMode: 0554
+      initContainers: #TODO: merge into one initContainer
+        - name: etcd-password-creator
+          image: quay.io/openshift/origin-cli
+          command: ["/bin/bash", "-c", "--"]
+          args:
+            - |
+              oc login --token=TOKEN --server=https://openshift.default.svc.cluster.local --insecure-skip-tls-verify
+              ETC_ROOT_PSW=$(cat /dev/urandom | tr -dc 'a-z0-9' | fold -w 32 | head -n 1)
+
+              echo "creating etcd passwords"
+
+              cat <<EOF | oc apply -f -
```

Comment:
Doesn't this result in the secret always being overwritten? Ideally we would:
1. Check if secret exists
2. If it doesn't, create the secret
Alternatively:
1. Always generate secret
2. Run `oc create`, which will fail if a secret exists. I don't really like this behaviour.

Reply:
updated initContainer logic to do nothing and exit with success if the secret already exists